src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -653,6 +653,42 @@ JNI_METHOD(void, pairDeviceWithAddress)
     }
 }
 
+JNI_METHOD(void, pairDeviceWithCode)
```

Comment:
Could you add one test for this API in https://github.com/project-chip/connectedhomeip/tree/master/examples/java-matter-controller/java/src/com/matter/controller/commands/pairing

Reply:
I added. (code, code-wifi, code-thread)